### ISSUE TITLE
symbols  comes multitime with multiple clicks

### ISSUE THREAD

**Issue by patson12** [frangoteam/FUXA#270]
Hi,,
i want to select any symbols and i want it to add in only one time. but it comes multiple time. like how many time i click in view editor it load everytime.. i want to load this symbol only one click and one time .. can you help me in this/?/
thanks in advance ..

**Comment by unocelli** [comment]
Hi,
sorry, but I don't think I understand what you're trying to do.

**Comment by patson12** [comment]
like if i select the circle from left panel and i drag into view windows then i should be only once .. but rightnow we can drag many times. so i want it for once only same with import svg and image..

**Comment by unocelli** [comment]
I’m closing this issue as there is currently no feasible solution and it is not a priority at this time. Thank you for the suggestion.
